File: fibonacci/fibonacci.swift
func generateFibonaccis(_ n: Int) -> [Int] {
    var fib = Array(repeating: 1, count: n)
    for i in 2..<n {
        fib[i] = fib[i - 2] + fib[i - 1]
    }
    return fib
}

var fib = generateFibonaccis(10)
print(fib.map({"\($0)"}).joined(separator: " "))

File: fibonacci/fibonacci.rb
def generate_fibonaccis(n)
    fib = [1, 1]
    (2...n).each do |i|
        fib[i] = fib[i - 1] + fib[i - 2]
    end
    fib
end

fib = generate_fibonaccis(10)
puts fib.join(" ")
